src/main/java/io/reactivex/exceptions/OnCompleteFailedException.java

```diff
@@ -18,7 +18,11 @@
     private static final long serialVersionUID = -6179993283427447098L;
 
     public OnCompleteFailedException(Throwable cause) {
-        super(cause);
+        super(cause != null ? cause : new NullPointerException());
```

Comment:
not sure it's a good idea, npe will point here and can confuse reader of the stacktrace.

// Also, creating npes manually is usually a bad practice, it's an exception thrown by VM, but RxJava throws them already so it's up to u